src/core/Tabs/Tabs.tsx

```diff
@@ -172,6 +194,7 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
 
     let newIndex = focusedIndex ?? currentActiveIndex;
     switch (event.key) {
+      case 'ArrowDown':
```

Comment:
Shouldn't we check for orientation too?

Reply:
I thought it would be ok to have both in both orientation, but [W3 disagrees](https://www.w3.org/TR/wai-aria-practices-1.1/#:~:text=if%20the%20tab%20list%20is%20horizontal%2C%20it%20does%20not%20listen%20for%20down%20arrow%20or%20up%20arrow%20so%20those%20keys%20can%20provide%20their%20normal%20browser%20scrolling%20functions%20even%20when%20focus%20is%20inside%20the%20tab%20list.).